global.json

```diff
@@ -1,18 +1,18 @@
 {
   "sdk": {
-    "version": "8.0.100-preview.1.23115.2",
+    "version": "8.0.301",
```

Comment:
The Arcade update brought us up to 8.0.101 but that version was giving a bad 'pass argument 1 as ref' diagnostic. Didn't see why we couldn't take a newer SDK.